src/index.test.ts

```diff
@@ -431,4 +431,21 @@ describe('ember-template-recast', function () {
       '{{foo-bar placeholder="Choose a \\"thing\\"..." p1="Choose a \\"thing\\"..."}}'
     );
   });
+
+  test('stack trace does not exceed', function () {
+    let b = builders;
+    let template = '<div @class="{{if foo "bar"}} baz" />';
```

Comment:
`@class` on a `<div>` seems pretty odd, let's use:

```suggestion
    let template = '<div class="{{if foo "bar"}} baz" />';
```